lib/utils.js

```diff
@@ -141,22 +142,26 @@ exports.GreatResponse = function (request, options, filterRules) {
 
     this.log = request.getLog();
 
-    if (options.requestHeaders) {
+    if (~options.requestHeaders.indexOf('response')) {
```

Comment:
For the sake of readability, just do the `-1` check; the next person might not understand what the code is doing.